doc/release-notes.md

```diff
@@ -94,8 +94,6 @@ will be removed in 18 weeks:
 - `legacy_privacy`
 - `getnewaddress`
 - `getrawchangeaddress`
-- `z_getbalance`
-- `z_gettotalbalance`
```

Comment:
`doc/book/src/user/deprecation.md` needs to be updated to logically revert the changes from https://github.com/zcash/zcash/commit/09cd65327b9b32b4992be020a71377f45eb0bdf6 (in practice, moving the two relevant lines from the Stage 2 table to the Stage 1 table.

Reply:
Thanks, fixed.